04-using-github-with-the-platform.Rmd

```diff
@@ -220,35 +224,41 @@ A Personal Access Token (PAT) is provided by GitHub through each user's profile,
   2. Select **_Generate new token_** and select the **scope** of access you want to grant using this token. **IMPORTANT: Unless you have specific reasons to need more control, just select the first group, `repo`--granting other rights can be dangerous if your PAT falls into the wrong hands, allowing someone else to irrevocably delete your repos, read and change your user profile settings, or even access billing information.**
   ![](images/pat/pat-scope.png)
   
-  3. Copy your PAT into a `.txt` file that you can keep somewhere securely. You won't be able to come back to this page to see this PAT again, and will have to generate a new one if you lose track of it.
+  3. Copy your PAT to the clipboard, for immediate use - see the next section.
+  
+     Note: GitHub won't show this PAT to you ever again BUT you shouldn't save it in a file or anything. If you were to have passwords in plain text lying around your hard disk then it becomes a security hazard. If you need a PAT in future (eg you wiped your .simply create a new one, and invalidate the previous one.
   ![](images/pat/pat-copy.png)
   
-  ### Using a PAT to Authenticate in R/RStudio
+### Using a PAT to Authenticate in R/RStudio
 
 A safe, fast, and reliable method is to store the PAT in the so-called **system environment**.
```

Comment:
Is there an obvious reason as to why storing the PAT in the system environment is less of a security risk than storing on your own local machine? In case people ask, rather than needing to reflect it here.

Reply:
The main thing we're fighting here is people committing the PAT into the repo. The reason is the 'need to know' principle. We've now mentioned that the PAT needs to be kept private to the user, and linked to the MoJ security guidance which talks more about this.